x/nexus/keeper/msg_server.go

```diff
@@ -13,7 +13,7 @@ import (
 
 var _ types.MsgServiceServer = msgServer{}
 
-const allChain = "*"
+const allChain = ":all:"
```

Comment:
`*` gives me headaches with bash scripts.

Reply:
It's not, just a workaround for now. A separate command might be preferable long term.